File: module15/module30/module1/src/main/java/com/android/example/module15_module30_module1/ClassAAE.java
package com.android.example.module15_module30_module1;

public class ClassAAE {
	private 	com.android.example.module15_module20_module1.ClassAAF instance_var_1_0 = new com.android.example.module15_module20_module1.ClassAAF();
	private 	com.android.example.module15_module20_module1.ClassAAD instance_var_1_1 = new com.android.example.module15_module20_module1.ClassAAD();
	private 	com.android.example.module15_module21_module1.ClassAAF instance_var_1_2 = new com.android.example.module15_module21_module1.ClassAAF();

	public void method0(
		com.android.example.module15_module21_module1.ClassAAG param0,
		com.android.example.module15_module21_module1.ClassAAE param1,
		com.android.example.module15_module20_module1.ClassAAI param2) throws Throwable {
		for (int iAb = 0; iAb < 1; iAb++) {
			param0.method0("SomeString", "SomeString", "SomeString");

			java.util.Collections.emptyList().forEach( lambda0 -> {
				try {
					param0.method0("SomeString", "SomeString", "SomeString");

					param0.method1("SomeString", "SomeString");

				} catch(Throwable e) { } // ignore
			});
			com.android.example.module15_module21_module1.ClassAAB local_var_3_0 = new com.android.example.module15_module21_module1.ClassAAB();
			local_var_3_0.method2("SomeString", "SomeString", "SomeString");

		}
	}

	public void method1(
		com.android.example.module15_module20_module1.ClassAAA param0,
		com.android.example.module15_module20_module1.ClassAAJ param1,
		com.android.example.module15_module21_module1.ClassAAE param2,
		com.android.example.module15_module20_module1.ClassAAJ param3) throws Throwable {
		param0.method0(new com.android.example.module15_module21_module1.ClassAAI(), new com.android.example.module15_module21_module1.ClassAAB(), new com.android.example.module15_module21_module1.ClassAAE(), new com.android.example.module15_module21_module1.ClassAAD());

		param0.method3(new com.android.example.module15_module21_module1.ClassAAF(), new com.android.example.module15_module21_module1.ClassAAE(), new com.android.example.module15_module21_module1.ClassAAC());

	}
}

File: module04/module03/module3/src/main/java/com/android/example/module04_module03_module3/ClassAAE.kt
package com.android.example.module04_module03_module3

@Suppress("UNUSED_PARAMETER")
class ClassAAE {
	private 	val instance_var_1_0: com.android.example.module04_module12_module6.ClassAAH = com.android.example.module04_module12_module6.ClassAAH()
	private 	val instance_var_1_1: com.android.example.module04_module12_module6.ClassAAI = com.android.example.module04_module12_module6.ClassAAI()
	private 	val instance_var_1_2: com.android.example.module04_module12_module6.ClassAAE = com.android.example.module04_module12_module6.ClassAAE()
	private 	val instance_var_1_3: com.android.example.module04_module12_module6.ClassAAG = com.android.example.module04_module12_module6.ClassAAG()
	private 	val instance_var_1_4: com.android.example.module04_module12_module6.ClassAAJ = com.android.example.module04_module12_module6.ClassAAJ()

	fun method0(
		param0: com.android.example.module04_module12_module6.ClassAAF,
		param1: com.android.example.module04_module12_module6.ClassAAC,
		param2: com.android.example.module04_module12_module6.ClassAAB,
		param3: com.android.example.module04_module12_module6.ClassAAA
	) {
		val local_var_2_4: io.reactivex.observers.TestObserver<kotlin.Any> = io.reactivex.observers.TestObserver()
		local_var_2_4.isCancelled()

		listOf("1", "2", "3").forEach {
			if (kotlin.Any().equals(kotlin.Any())) {
				val local_var_4_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
				local_var_4_0.cancel()

			}
			val local_var_3_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
			local_var_3_0.onNext(kotlin.Any())

		}
		val local_var_2_5: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_5.cancel()

		param0.method1("SomeString", "SomeString", "SomeString")

	}

	fun method1(
		param0: com.android.example.module04_module12_module6.ClassAAF,
		param1: com.android.example.module04_module12_module6.ClassAAG,
		param2: com.android.example.module04_module12_module6.ClassAAG
	) {
		val local_var_2_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_3.onComplete()

	}

	fun method2(
		param0: com.android.example.module04_module12_module6.ClassAAB,
		param1: com.android.example.module04_module12_module6.ClassAAE,
		param2: com.android.example.module04_module12_module6.ClassAAA,
		param3: com.android.example.module04_module12_module6.ClassAAH
	) {
	}

	fun method3(
		param0: com.android.example.module04_module12_module6.ClassAAH,
		param1: com.android.example.module04_module12_module6.ClassAAA
	) {
		param0.method0("SomeString", "SomeString")

	}
}
